Assets/Scripts/ArenasParameters.cs

```diff
@@ -213,7 +215,6 @@ public void Add(int k, YAMLDefs.Arena yamlConfig)
                 }
             }
             CurrentArenaID = k;
-            yamlConfig.SetCurrentPassMark();
```

Comment:
Sorry, having some trouble getting my head around what was broken - was this just looping through all the arena configs at the top and overwriting the passMark each time? (So i.e. for the whole run this is set to the passMark of the final arena?)

Reply:
Not quite. The passMark was simply not being set individually per arena, regardless of what ever number of arenas were set. It would also set the first arena passMark value to 0 (this is the default value). However, not every arena had their passMark value incorrectly set. It's just that there is erratic behaviour and inconsistencies on how the passMark was being set and which arena config it was assigned to.

Reply:
> that there is erratic behaviour and inconsistencies on how the passMark was being set

What was the behaviour, and what were the inconsistencies?